core/src/Alert.html

```diff
@@ -73,7 +72,8 @@
 
         return Object.entries(links).reduce((acc, [key, content]) => {
           const elemId = key;
-          const processedData = `<a id="${elemId}">${content.text}</a>`;
+          const escapedText = sanitizeHtml(content.text);
+          const processedData = `<a id="${elemId}">${escapedText}</a>`;
           const pattern = new RegExp(`({${key}})`, 'g');
```

Comment:
The key can contain characters that regexp interpretes (like '.' for example). The key needs to be passed here regexp-escaped! (See here https://stackoverflow.com/questions/3561493/is-there-a-regexp-escape-function-in-javascript/3561711).